pom.xml

```diff
@@ -356,9 +355,51 @@
                                 <exclude>checkstyle:/src/test/java/org/takes/facets/fork/TkMethodsTest.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/StateAwareInputStream.java</exclude>
                                 <exclude>checkstyle:/src/test/java/org/takes/misc/VerboseListTest.java</exclude>
+
+                                <!--
+                                @todo #517:30min Continue fixing FindBugs violations started in #517.
+                                 Major bugs found in these classes are
+                                 Synchronization performed on java.util.concurrent.CopyOnWriteArrayList
+                                 refactored into a named _static_ inner class
```

Comment:
@sebing for me it is confusing. Let's do this puzzle as follows:
```
@todo #517:30min Continue fixing FindBugs violations started in #517
 and remove all excludes below.
```
This should be enough to understand what needs to be done.